From 420cadec0f7a6048a0fafccf4c658d543b7dbf19 Mon Sep 17 00:00:00 2001
From: Koen Kooi <k-kooi@ti.com>
Date: Tue, 26 Jul 2011 14:25:39 +0200
Subject: [PATCH] matrix_gui.pro: add dbus to QT features to fix linking with  -Wl,--as-needed

This fixes:

| ccache arm-angstrom-linux-gnueabi-g++ -march=armv7-a -mtune=cortex-a8 -mfpu=neon -mfloat-abi=softfp -fno-tree-vectorize -mthumb-interwork -mno-thumb --sysroot=/home/koen/angstrom-core/build/tmp-angstrom_2010_x-eglibc/sysroots/beagleboard -Wl,-O1 -Wl,--hash-style=gnu -Wl,--as-needed -Wl,-rpath-link,/home/koen/angstrom-core/build/tmp-angstrom_2010_x-eglibc/sysroots/beagleboard/usr/lib -o matrix_gui main.o mainwindow.o buttonfactory.o buttonlaunch.o customstyle.o menupage.o osdmanager.o matrixcontrol.o moc_mainwindow.o moc_buttonfactory.o moc_buttonlaunch.o moc_customstyle.o moc_menupage.o moc_osdmanager.o moc_matrixcontrol.o    -L/home/koen/angstrom-core/build/tmp-angstrom_2010_x-eglibc/sysroots/beagleboard/usr/lib -lpng -lfreetype -lz -lgthread-2.0 -lQtWebKit -lQtGui -lQtNetwork -lQtCore -lpthread
| ux-gnueabi/4.5.4/ld: }: invalid DSO for symbol `QDBusConnection::registerService(QString const&)' definition
| /home/koen/angstrom-core/build/tmp-angstrom_2010_x-eglibc/sysroots/beagleboard/usr/lib/libQtDBus.so.4: could not read symbols: Bad value
| collect2: ld returned 1 exit status

Signed-off-by: Koen Kooi <k-kooi@ti.com>
---
 matrix_gui.pro |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/matrix_gui.pro b/matrix_gui.pro
index 5463310..908e8a7 100644
--- a/matrix_gui.pro
+++ b/matrix_gui.pro
@@ -1,4 +1,5 @@
-QT += webkit \
+QT += dbus \ 
+    webkit \
     network
 LIBS += -lpng -lfreetype -lz -lgthread-2.0
 HEADERS = mainwindow.h \
-- 
1.7.4.4

